packages/playwright/src/mcp/program.ts

```diff
@@ -76,6 +79,11 @@ export function decorateCommand(command: Command, version: string) {
         }
 
         const config = await resolveCLIConfig(options);
+
+        // Chromium browsers require ffmpeg to be installed to save video.
+        if (config.saveVideo)
```

Comment:
Let the users do it manually or at least print a message to stderr that we are dowloading something?